recipes/jupyter-desktop-server/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "jupyter-desktop-server" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyter-desktop-server-{{ version }}.tar.gz
+  sha256: 5915a2dbbe250f59b3cc01859297b26b2e1836e03a5db4bfbfd438005649d7e4
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - jupyter-server-proxy >=1.4.0
+    - python >=3.8
+
+test:
+  imports:
+    - jupyter_desktop
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/jupyter-desktop-server/
+  summary: Run a Linux Desktop on a JupyterHub
+  license: BSD-3-Clause
+  license_file: jupyter_desktop/share/web/noVNC-1.1.0/LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - yuvipanda
+    - weiji14
```

```suggestion
    - weiji14
    - manics
```
I'm happy to co-maintain this.